lms/djangoapps/teams/static/teams/js/spec/views/edit_team_spec.js

```diff
@@ -13,21 +13,21 @@ define([
         var teamsUrl = '/api/team/v0/teams/',
             createTeamData = {
                 id: null,
-                name: "TeamName",
-                course_id: "a/b/c",
-                topic_id: "awesomeness",
-                date_created: "",
-                description: "TeamDescription",
-                country: "US",
-                language: "en",
+                name: 'TeamName',
```

Comment:
Nit: personally I don't care about mixing single and double quotes, and switching existing quotes makes reviews harder. My preference would be not to change them unnecessarily (not that you need to change what you've already done in this PR... a suggestion for going forward).